vm/vmimpl/linux.go

```diff
@@ -14,6 +14,6 @@ import (
 const (
 	unix_CBAUD     = unix.CBAUD
 	unix_CRTSCTS   = unix.CRTSCTS
-	syscall_TCGETS = syscall.TCGETS
-	syscall_TCSETS = syscall.TCSETS
+	syscall_TCGETS = unix.TCGETS2
```

Comment:
I think build will now fail with "unused package syscall".
You can test by doing `GOOS=linux go install ./...`